content/learn/03.programming/05.arduino-and-python/arduino-and-python.md

```diff
@@ -1,17 +1,17 @@
 ---
-author: 'Karl Söderby'
-title: 'MicroPython with Arduino Boards'
-description: 'Learn about compatibility between the popular MicroPython implementation and Arduino boards, how to set up your environment, and discover in-depth tutorials and useful links.'
-tags: 
+author: "Karl Söderby"
+title: "MicroPython with Arduino Boards"
+description: "Learn about compatibility between the popular MicroPython implementation and Arduino boards, how to set up your environment, and discover in-depth tutorials and useful links."
+tags:
   - MicroPython
   - Arduino Lab
   - OpenMV
 hero_position: 1
 ---
 
-***To download the firmware required to run MicroPython on your Arduino board, visit the [Arduino MicroPython downloads page](/micropython).***
+**_To download the firmware required to run MicroPython on your Arduino board, visit the [Arduino MicroPython downloads page](/micropython)._**
```

Comment:
@Matteo-it why was this changed? This should still be three *** since this is what creates the "note"-tags.

Reply:
There are other changes in this file as well where this change was implemented where it should not :(

Reply:
Sorry @jhansson-ard, i was testing the new CodeBlock in this page, i will replace this file with the main version soon :D